delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutesSpec.scala

```diff
@@ -394,6 +395,27 @@ class ResourcesRoutesSpec extends BaseRouteSpec {
       }
     }
 
+    "validate a resource successfully against its latest schema" in {
+      Get("/v1/resources/myorg/myproject/_/myid2/validate") ~> routes ~> check {
+        status shouldEqual StatusCodes.OK
+        response.asJson should have(`@type`("sh:ValidationReport"))
+      }
+    }
+
+    "validate a resource against a schema that does not exist" in {
+      Get("/v1/resources/myorg/myproject/pretendschema/myid2/validate") ~> routes ~> check {
+        status shouldEqual StatusCodes.NotFound
+        response.asJson should have(`@type`("InvalidSchemaRejection"))
```

Comment:
I think it is important not to check only for the `@type` and make sure the details are also properly serialized